Packs/AzureWAF/Integrations/AzureWAF/AzureWAF.py

```diff
@@ -332,6 +345,23 @@ def reset_auth(client: AzureWAFClient):
                                           '**!azure-waf-auth-start** and **!azure-waf-auth-complete**.')
 
 
+@logger
+def test_module(client):
+    """
+    Performs basic GET request to check if the API is reachable and authentication is successful.
+    Returns ok if successful.
+    """
+    if demisto.params().get('auth_type') == 'Device':
```

Comment:
we cant run azure-waf-auth-test here?

Reply:
No, You need to configure the instance first.. we already tried that